src/gluonts/ev_v3/metrics.py

```diff
@@ -0,0 +1,402 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+from typing import Optional, Union
+
+import numpy as np
+
+from gluonts.ev_v3.api import BaseMetric, AggregateMetric, get_standard_type
+from gluonts.model.forecast import Quantile
+from gluonts.time_feature import get_seasonality
+
+
+# --- INPUTS & BASE METRICS (same dimensionality as inputs) ---
+
+# allows using `PredictionTarget()` instead of `data["PredictionTarget"]`
+class PredictionTarget(BaseMetric):
+    def get_name(self):
+        return "PredictionTarget"
+
+    def calculate(self, data):
+        assert self.name in data, f"Missing input '{self.name}' in data"
```

Comment:
To let the user now that `data` does not contain the required keys. Looking at this now, it's quite useless because the thrown KeyError would have said pretty much the same. A GluonTSUserError might be suitable here.